ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -16,90 +12,77 @@ base_images:
     namespace: ocp
     tag: "8"
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: rhel-8-release-golang-1.19-openshift-4.13
+  project_image:
+    dockerfile_literal: |
+      FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.19-openshift-4.13
+      USER root
+      RUN echo -e '[google-cloud-sdk]\n\
+      name=Google Cloud SDK\n\
```

Comment:
Build root shouldn't be opinionated about cloud provider.  It'd be better to do this in a follow on `.images[]` image.

Reply:
More generally, the build image should eventually be moved to the microshift repo, and should be runnable locally to build rpms.  So it should only contain the bare minimum needed to meet that req.

Reply:
> I'd say if we want to build RPMs in container, something not behind auth would be better and more confortable to use.

Fair point, I'd forgotten about that aspect when I wrote that comment. 

> As a developer, why would I want to use a container to build the RPM? For consistency with CI?

I was thinking more for a consistent build env amoung devs.  But yeah, the dependency on the pull-secret makes that less convenient that I was hoping it'd be.